components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/listener/OAuthCacheRemoveListener.java

```diff
@@ -53,18 +53,18 @@ public void entryRemoved(CacheEntryEvent<? extends OAuthCacheKey, ? extends Cach
             log.debug("OAuth cache removed for consumer id : " + accessTokenDO.getConsumerKey());
         }
 
+        String userName = accessTokenDO.getAuthzUser().toString();
         boolean isUsernameCaseSensitive = IdentityUtil
                 .isUserStoreInUsernameCaseSensitive(accessTokenDO.getAuthzUser().getUserName());
```

Comment:
Please send the username retrieved from  accessTokenDO.getAuthzUser().toString() to  isUserStoreInUsernameCaseSensitive method.